app/src/main/java/com/example/mealsnap_prototype_1/ConfigActivity.java

```diff
@@ -0,0 +1,33 @@
+package com.example.mealsnap_prototype_1;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+
+import services.authhandler.Amplify;
+
+public class ConfigActivity extends AppCompatActivity {
+
+    //Config activity will config stuff we need to(passes or succeeds activity just redirects with intents to what we wanna check).
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_config);
+
+        //Test Amplify Config
+        Amplify testAmplify = new Amplify();
+        testAmplify.configure(getApplicationContext());
```

Comment:
since `configure` is a static method, initialization of Amplify is not needed.

Reply:
Acknowledged in recent push